api-catalog-ui/frontend/src/components/GraphQL/GraphQLUIApiml.jsx

```diff
@@ -31,11 +33,14 @@ export default function GraphQLUIApiml(props) {
             body: JSON.stringify(graphQLParams),
```

Comment:
I believe it'd be better to add a try catch, in case the rest call would fail. Something like:
```javascript
const fetcher = async (graphQLParams) => {
        try {
            const response = await fetch(basePath, {
                method: 'POST',
                headers: {
                    Accept: 'application/json',
                    'Content-Type': 'application/json',
                },
                body: JSON.stringify(graphQLParams),
                credentials: 'same-origin',
            });
            const data = await response.clone().json().catch(() => response.text());
            return data;
        } catch (error) {
            console.error('Error fetching data:', error);
            return { errors: [{ message: 'Failed to fetch data' }] };
        }
    };
```